helmfile/charts/astrago/templates/inference/deployment.yaml

```
@@ -0,0 +1,96 @@
+      {{- end }}
+      {{- with .Values.inference.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.inference.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      terminationGracePeriodSeconds: 30
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (2 > 0)

(empty-lines)